changes/issue-5408-transparency-url

```diff
@@ -1,4 +1,3 @@
 - Add `fleet_desktop.transparency_url` to `app_config_json`
 - Set default `transparency_url="https://fleetdm.com/transparency`
 - Enable Fleet Premium licensees to set custom `transparency_url` via REST API and `fleetctl apply`
-- Add `transparency_url` to `GET /device/{token}` endpoint response
```

Comment:
- Add the redirect API etc.

Reply:
didn't add this here because we didn't wanted to document device endpoints at all in the past (not even on release notes). Should I?

Reply:
Good point.